pkg/rootless/rootless_linux.c

```diff
@@ -263,6 +263,13 @@ do_preexec_hooks_dir (const char *dir, char **argv, int argc)
 static void
 do_preexec_hooks (char **argv, int argc)
 {
+  // Access the preexec_hooks_dir indicator file
+  // return without processing if the file doesn't exist
+  char preexec_hooks_path[] = "/etc/containers/podman_preexec_hooks.txt";
```

Comment:
Implementation LGTM. (I’m not sure why this needs to exist, looking just at the stated intent.)

---

(This isn’t really worth a delay in _this_ PR, but if we are going to add similar code to the main branch…:)

- Stylistically, putting a macro at the top, beside `ETC_PREEXEC_HOOKS`, would make it clearer that this is a ~choice, not an essential component of this individual function.
- It’s fine to call `access("…")` without the intermediate `char[]` variable.